src/components/Company.js

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { fetchStockScreener, selectCompany } from '../redux/stockScreener/StockScreenersSlice';
+import Search from './Search';
+import Footer from './Footer';
+import './styles/Company.css';
+
+function Company() {
+  const dispatch = useDispatch();
+  const stockScreeners = useSelector((state) => state.stockScreeners);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setIsLoading(true);
+      dispatch(fetchStockScreener());
+      setIsLoading(false);
+    };
+    fetchData();
+  }, [dispatch]);
+
+  return (
+    <div className="navigation">
+      <ul className="nav-bar">
+        <li className="subject">Stock Screener</li>
+        <li>
+          <Search />
+        </li>
+      </ul>
+      <div className="companyContainer">
+        {isLoading && <p>Loading...</p>}
+        {Array.isArray(stockScreeners.searchStockCompany)
+          && stockScreeners.searchStockCompany.map((screener) => (
+            <div className="company-card" key={screener.symbol}>
+              <Link
+                to="/Screeners"
+                onClick={() => dispatch(selectCompany(screener))}
+              >
+                <ul className="company-list" key={screener.symbol}>
+                  <li
+                    className="companySymbol"
+                    style={{
+                      fontFamily: 'stencil Std, fantasy',
+                      fontSize: '24px',
+                    }}
+                  >
+                    {screener.symbol}
+                  </li>
+                  <li style={{ marginBottom: '50px' }}>{screener.country}</li>
+                  <li style={{ fontFamily: 'Chalkduster, fantasy' }}>
+                    {screener.companyName}
+                  </li>
+                  <li>{screener.sector}</li>
+                </ul>
+              </Link>
+            </div>
+          ))}
+      </div>
+      <Footer />
+    </div>
```

Comment:
- [ ] Kindly note that according to project requirements, you are to keep to typography, spacing and generally the layouts, currently Your cards are stacked in a one-column grid in the mobile view, but the design uses a two-column grid, also There is no space between cards in the given design while yours have spaces between, kindly fix this
your design :point_down: 
![Screenshot from 2023-09-06 11-03-30](https://github.com/Bhekisile/Metrics-webapp/assets/66097371/64454fc3-cfd1-4093-af6e-bcb435ce7a46)
given design :point_down: 
![Screenshot from 2023-04-28 11-04-25](https://github.com/Bhekisile/Metrics-webapp/assets/66097371/285a35a0-1bed-466b-9352-8087a819e5a1)